extensions/amp-facebook-comments/0.1/amp-facebook-comments.js

```diff
@@ -46,7 +46,7 @@ class AmpFacebookComments extends AMP.BaseElement {
     this.preconnect.url('https://facebook.com', opt_onLayout);
     // Hosts the facebook SDK.
     this.preconnect.preload(
-        'https://connect.facebook.net/en_US/sdk.js', 'script');
+        'https://connect.facebook.net/' + window.navigator.language.replace('-', '_') + '/sdk.js', 'script');
```

Comment:
Where can we create this helper? Can we translate `navigator.language` pt-BR to `navigator.locale` pt_BR?
```
navigator.locale = navigator.language.replace('-', '_')
```

or
```
function toLocale(str){  return str.replace('-', '_') }
```